src/impact-dashboard/ChartArea.tsx

```diff
@@ -45,10 +46,12 @@ const ChartArea: React.FC<{
 
   const chartData = useChartDataFromProjectionData(projectionData);
 
+  const title = alternateTitle ? "Estimated Impact" : "Projection";
```

Comment:
this is OK but maybe would be more flexible to just make `title` an argument with a default value?